CONTRIBUTING.md

```diff
@@ -25,7 +35,7 @@ Thanks for contributing!
 
 ### Code style
 
-The coding style suggested by the Golang community is used in operator-sdk. See the [style doc][golang-style-doc] for details.
+The coding style suggested by the Golang community is used in operator-sdk. See the [style doc](https://google.github.io/styleguide/javaguide.html) for details.
```

Comment:
Since we are using the java guide, lets change this to something along the lines of "...by the Java community is used in the Java Operator SDK"

Reply:
Modified. Thank you for you comments.